src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java

```diff
@@ -64,8 +64,9 @@ void transferChromeOptionPreferencesFromSystemPropsToDriverNoAssignmentStatement
     ChromeOptions chromeOptions = new ChromeDriverFactory().createChromeOptions(config, proxy);
     Map<String, Object> prefsMap = getBrowserLaunchPrefs(ChromeOptions.CAPABILITY, chromeOptions);
 
-    assertThat(prefsMap).hasSize(2);
+    assertThat(prefsMap).hasSize(3);
```

Comment:
I think the lines 67,69,70 are "tests that are bound to implementation", which should be avoided in general.

Reply:
I agree. We could check not the exact size, but onle the needed properties.